server/src/main/java/org/opensearch/action/admin/cluster/snapshots/restore/RestoreSnapshotRequest.java

```diff
@@ -148,6 +149,9 @@ public RestoreSnapshotRequest(StreamInput in) throws IOException {
         if (in.getVersion().onOrAfter(Version.V_2_7_0)) {
             storageType = in.readEnum(StorageType.class);
         }
+        if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
```

Comment:
can we use `V_2_9_0` everywhere ?  This would make backporting easier . bwc tests can fail till we port it to 2.9. But we should be backporting pretty quickly after main merge .

Reply:
makes sense. this looks like a better option, will update the version checks to use 2.9.

Reply:
will update this with a separate PR actually. otherwise it would be hard to figure if the failures are happening only due to this.